### ISSUE TITLE
Cold Staking: A generated hot addresses can be confused with a normal address 

### ISSUE THREAD

**Issue by thecrypt0hunter** [block-core/blockcore#23]
Because a hot address is formatted exactly the same as a normal spending address, it is very easy for a user to become confused and send funds directly to the hot address. Whilst it's possible to retrieve these funds (with a bit of manual effort) ... it would be preferable that the hot address is easily differentiated (different first character or different length) and a UI could stop funds being sent directly to this address.

**Comment by dangershony** [comment]
I think to solve this we could generate a different bech32 prefix say an address starting with `odxcs` instead of just `odx` this should be possible if we add another bech32 encoder

**Comment by thecrypt0hunter** [comment]
Can the address be made longer? So not just a change to first few characters ... It would be possible to send coins to that address whereas most wallets already check the length of the address.

**Comment by dangershony** [comment]
Yes but it will be some custom solution, I think that just the prefix will be enough (it will be longer anyway)

**Comment by thecrypt0hunter** [comment]
In that case could you flip the prefix from `odxcs` to `csodx` - it's more clearly not a normal address
